### ISSUE TITLE
[Feature Request] Add song to mpd/mpc

### ISSUE THREAD

**Issue by lionawurscht** [mps-youtube/yewtube#168]
Hey,
I just discovered your programm and love it. Unfortunately I also started using mpd a short while ago. Mpd can also play streams so it would theoretically possible to implement the option of adding the songlist to mpd (eg via mpc add) so they will be played later. This would allow me to play both local files and youtube songs in the same playing queue.

**Comment by ids1024** [comment]
Players other than mpv and mplayer are not supported, but this works for me:

    set player mpc
    set playerargs add

**Comment by ata2001** [comment]
> Players other than mpv and mplayer are not supported
Would you accept it, if someone implemented mpc like the other players?

**Comment by ids1024** [comment]
> Would you accept it, if someone implemented mpc like the other players?

Yes, that would be reasonable. But `player.py` is currently a bit of a mess; it would probably be better to first refactor it to be class-based (a `Player` class, with `Mpv`, `Mplayer`, etc. classes derived from it).

**Comment by ata2001** [comment]
Should we create a new issue then?

**Comment by BobSpams** [comment]
The "mpc add" isn't working for me
I have mpc, mpd installed
